tests/drivers/build_all/sensor/spi.dtsi

```diff
@@ -337,3 +337,8 @@ test_spi_icm426888: icm42688@28 {
 	gyro-hz = <32000>;
 	gyro-fs = <2000>;
 };
+
+test_spi_akm09918c: akm09918c@29 {
```

Comment:
i2c device correct? seems like the wrong file and dts node here

Reply:
Oops :)